server/skillcatalog/catalog.go

```
@@ -0,0 +1,373 @@
+		inspections = append(inspections, inspection)
+		if inspection.Loaded {
+			skills = append(skills, Skill{Name: inspection.Name, Description: inspection.Description, Path: inspection.Path, SourceKind: root.Kind})
+		}
+	}
+	return inspections, skills, issues, nil
+}
+
+func discoverEmbeddedGenerated(projectedRoot string) ([]Inspection, []Skill, []Issue, error) {
+	entries, err := fs.ReadDir(prompts.GeneratedSkillsFS, SkillsDirName)
+	if err != nil {
+		return nil, nil, []Issue{{SourceKind: SourceKindGenerated, Reason: err.Error()}}, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 222) but it returns nil

(nilerr)